common/test/acceptance/tests/studio/test_studio_home.py

```diff
@@ -173,3 +178,78 @@ def test_studio_language_change(self):
             get_selected_option_text(language_selector),
             u'Dummy Language (Esperanto)'
         )
+
+
+class CourseNotEnrollTest(WebAppTest):
+    """
+    Test that we can create a new content library on the studio home page.
+    """
+
+    def setUp(self):
+        """
+        Load the helper for the home page (dashboard page)
+        """
+        super(CourseNotEnrollTest, self).setUp()
+
+        self.auth_page = AutoAuthPage(self.browser, staff=True)
+        self.dashboard_page = DashboardPage(self.browser)
+        self.course_name = "New Course Name"
+        self.course_org = "orgX"
+        self.course_number = str(uuid.uuid4().get_hex().upper()[0:6])
+        self.course_run = "2015_T2"
+
+    def course_id(self):
+        """
+        Returns the serialized course_key for the test
+        """
+        # TODO - is there a better way to make this agnostic to the underlying default module store?
+        default_store = os.environ.get('DEFAULT_STORE', 'draft')
+        course_key = CourseLocator(
+            self.course_org,
+            self.course_number,
+            self.course_run,
+            deprecated=(default_store == 'draft')
+        )
+        return unicode(course_key)
+
+    def test_unenroll_course(self):
+        """
+        From the home page:
```

Comment:
@attiyaIshaque , I don't think we actually need a bok-choy test for this PR, since we can get all the information we need to test in unit tests.